src/vm/runtimehandles.cpp

```diff
@@ -1373,17 +1373,6 @@ FCIMPL2(FC_BOOL_RET, RuntimeTypeHandle::CanCastTo, ReflectClassBaseObject *pType
     {
         iRetVal = (r == TypeHandle::CanCast);
     }
-
-    // We allow T to be cast to Nullable<T>
-    if (!iRetVal && Nullable::IsNullableType(toHandle) && !fromHandle.IsTypeDesc())
```

Comment:
Where did this part of the logic went?

Reply:
the case is handled above  -  fromHandle.CanCastTo   will always return true   when  `!fromHandle.IsTypeDesc()`,  this was dead code



---
In reply to: [335150545](https://github.com/dotnet/coreclr/pull/23548#discussion_r335150545) [](ancestors = 335150545)